README.md

```diff
@@ -11,31 +11,70 @@ of the model. This results in speedups (without any additional code optimization
 The code is arranged as a package `slicegpt` in `/src`, and scripts to replicate experiments from the paper are in 
 `/experiments`. To install the `slicegpt` package, we recommend
 
-`pip install -e .`
+```
+    pip install -e . 
+```
 
 ## Running SliceGPT
 
 To run SliceGPT on `microsoft/phi-2`, from the `experiments` folder, run 
 ```
-    python run_slicegpt_perplexity.py \
+    python run_slicegpt.py \
            --model microsoft/phi-2 \
            --save-dir dir/to/save/sliced_model/in \
            --sparsity 0.25 \
-           --no-wandb \
            --device cuda:0 \
-           --eval-baseline
+           --eval-baseline \
+           --no-wandb
 ```
 
 This will compress the `microsoft/phi-2` model and save the compressed model to the specified directory. Please consult 
 the script for the full set of options.
 
-The experiments folder also contains scripts for 
-- [finetuning](./experiments/run_finetuning.py) the compressed model to recover most of the quality lost during compression
-- [zero-shot task evaluation](./experiments/run_zero_shot_tasks.py) of a dense, compressed or fine-tuned model
-
 _Note:_ For models that require Hugging Face authentication, set the `--hf-token` argument 
 manually or using a key vault. Alternatively, set the environment variable `HF_TOKEN`.
 
+### Recovery fine-tuning
+
+To install additional dependencies required for post-slicing recovery fine-tuning (RFT):
+
+```
+    pip install -e .[finetune]
+```
+
+The following replicates the experiments in the paper (LoRA hyperparams valid for all Llama-2 and Phi-2 models): 
```

Comment:
Have you tried running this with the fixes that I added in my PR #101 yet? I expect better results than in the paper

Reply:
We _should_ do this, but it can come as a separate change, with another update to the readme if needed.